test/unit/utils/isFormData.js

```diff
@@ -0,0 +1,12 @@
+var assert = require('assert');
+var isFormData = require('../../../lib/utils').isFormData;
+var FormData = require('form-data');
+
+describe('utils::isFormData', function () {
```

Comment:
Should we add a test for web as well?

Reply:
Axios already has it. https://github.com/axios/axios/blob/c9aca7525703ab600eacd9e95fd7f6ecc9942616/test/specs/utils/isX.spec.js#L26